### ISSUE TITLE
random doctest failure in src/doc/en/thematic_tutorials/group_theory.rst

### ISSUE THREAD

**Issue by yyyyx4** [sagemath/sage#34901]
<div id="comment:0"></div>

Part of #32544:

```
sage -t --long --random-seed=185470747385175316039405094141387664605 src/doc/en/thematic_tutorials/group_theory.rst
**********************************************************************
File "src/doc/en/thematic_tutorials/group_theory.rst", line 207, in doc.en.thematic_tutorials.group_theory
Failed example:
    euler_phi(m*n) == euler_phi(m) * euler_phi(n)
Expected:
    True
Got:
    False
**********************************************************************
1 item had failures:
   1 of 170 in doc.en.thematic_tutorials.group_theory
    [127 tests, 1 failure, 4.40 s]
```

(From a patchbot run in #34880.)

Component: **documentation**

Author: **Frédéric Chapoton**

Branch/Commit: **[u/chapoton/34901](https://github.com/sagemath/sagetrac-mirror/tree/u/chapoton/34901) @ [`a8ee692`](https://github.com/sagemath/sagetrac-mirror/commit/a8ee69222295df9eb3d17e2ec1804246824e4f55)**

Reviewer: **Lorenz Panny**

_Issue created by migration from https://trac.sagemath.org/ticket/34901_

**Comment by yyyyx4** [comment]
<div id="comment:1" align="right">comment:1</div>

In the test, `m` and `n` are random primes less than `10000`. The test fails if and only if `m == n`.

**Comment by fchapoton** [comment]
<div id="comment:2"></div>

New commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/a8ee69222295df9eb3d17e2ec1804246824e4f55"><code>a8ee692</code></a></td><td><code>fix a random doctest in tutorial</code></td></tr></table>

**Comment by fchapoton** [comment]
Branch: **[u/chapoton/34901](https://github.com/sagemath/sagetrac-mirror/tree/u/chapoton/34901)**

**Comment by fchapoton** [comment]
Commit: **[`a8ee692`](https://github.com/sagemath/sagetrac-mirror/commit/a8ee69222295df9eb3d17e2ec1804246824e4f55)**

**Comment by fchapoton** [comment]
Author: **Frédéric Chapoton**

**Comment by fchapoton** [comment]
<div id="comment:3" align="right">comment:3</div>

ok like that ?

**Comment by yyyyx4** [comment]
<div id="comment:4" align="right">comment:4</div>

That should do. Thanks!

**Comment by yyyyx4** [comment]
Reviewer: **Lorenz Panny**

**Comment by mkoeppe** [comment]
Merged in 10.0.beta0
